packages/i18n/src/locales/en.i18n.json

```diff
@@ -5792,6 +5792,8 @@
   "view-all-teams_description": "Permission to view all teams",
   "view-all-team-channels": "View All Team Channels",
   "view-all-team-channels_description": "Permission to view all team's channels",
+  "view-members-list-all-rooms": "Can view members in all rooms",
+  "view-members-list-all-rooms_description": "Gives the ability to see the members list in all rooms, even those the user is not part of",
```

Comment:
Not requesting any specific change, but I just wanted to confirm whether this has been approved by our UX/UI team. If not, it'd be good to have their take on this too (otherwise you can resolve this :slightly_smiling_face: )

Reply:
Yep, there's actually a comment by me some lines above